ddtrace/internal/remoteconfig/worker.py

```diff
@@ -15,7 +15,18 @@
 
 def get_poll_interval_seconds():
     # type:() -> float
-    return float(os.getenv("DD_REMOTECONFIG_POLL_SECONDS", default=DEFAULT_REMOTECONFIG_POLL_SECONDS))
+    if os.getenv("DD_REMOTECONFIG_POLL_SECONDS"):
+        log.warning(
+            "DD_REMOTECONFIG_POLL_SECONDS environment variable is deprecated in favor of "
+            "DD_REMOTE_CONFIG_POLL_INTERVAL_SECONDS.\n"
+            "DD_REMOTECONFIG_POLL_SECONDS will be removed in next version"
+        )
+    return float(
+        os.getenv(
+            "DD_REMOTE_CONFIG_POLL_INTERVAL_SECONDS",
+            default=os.getenv("DD_REMOTECONFIG_POLL_SECONDS", default=DEFAULT_REMOTECONFIG_POLL_SECONDS),
+        )
+    )
```

Comment:
We should use `ddtrace.vendor.debtcollector.deprecate()` instead of logging a warning.

```suggestion
    if os.getenv("DD_REMOTECONFIG_POLL_SECONDS"):
        deprecate(
             "Using environment variable 'DD_REMOTECONFIG_POLL_SECONDS' is deprecated",
             message="Please use DD_REMOTE_CONFIG_POLL_INTERVAL_SECONDS instead.",
             removal_version="2.0.0",
        )

    return float(
        os.getenv(
            "DD_REMOTE_CONFIG_POLL_INTERVAL_SECONDS",
            default=os.getenv("DD_REMOTECONFIG_POLL_SECONDS", default=DEFAULT_REMOTECONFIG_POLL_SECONDS),
        )
    )
```

Reply:
nice! :clap: